src/utils/event.ts

```diff
@@ -21,8 +21,13 @@ import {
 export const getCurrentEvent = (roomEvents: VenueEvent[]) =>
   roomEvents.find(isEventLive);
 
-export const isEventLive = (event: VenueEvent) =>
-  isWithinInterval(Date.now(), getEventInterval(event));
+export const isEventLive = (event: VenueEvent) => {
+  if (!event?.start_utc_seconds) {
+    return false;
+  }
```

Comment:
code used to crash if there were no `start_utc_seconds` for an event here, faced this problem on one of my venues, probably due to incorrect event data, but had to add that check just in case